lib/features/manage_account/domain/usecases/create_new_default_identity_interactor.dart

```diff
@@ -37,8 +37,9 @@ class CreateNewDefaultIdentityInteractor {
     final listIdentities = await _identityRepository
         .getAllIdentities(
             accountId, 
-            properties: Properties({'sortOrder'})
+            properties: Properties({'sortOrder', 'mayDelete'})
         );
+    listIdentities.identities?.removeWhere((identity) => identity.mayDelete != true);
```

Comment:
should wrap `(identity) => identity.mayDelete != true` in a method for more understand